### PR TITLE
Add crossfi-mainnet-1

### PR DESCRIPTION
**Pull request by MortelliGaming**
this is the exact copy of the official one ffrom 
https://xficonsole.com/

### CODE REVIEW

File: cosmos/crossfi-mainnet [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    {
+      "coinDecimals": 18,
+      "coinDenom": "MPX",
+      "coinMinimalDenom": "mpx"
+    }
+  ],
+  "features": [
+    "ibc-go",
+    "ibc-transfer",
+    "query:/cosmos/bank/v1beta1/spendable_balances",
```
Comment (editaahn) [inline cosmos/crossfi-mainnet:33]: This line is unnecessary.

**Review by editaahn — COMMENTED** [review]
Could you please update this file format extension to json?

File: cosmos/crossfi-mainnet (hunk 1 above)
Comment (MortelliGaming) [reply cosmos/crossfi-mainnet]: done

File: cosmos/crossfi-mainnet.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    {
+      "coinDecimals": 18,
+      "coinDenom": "MPX",
+      "coinMinimalDenom": "mpx"
+    }
+  ],
+  "features": [
+    "ibc-go",
+    "ibc-transfer",
```
Comment (editaahn) [inline cosmos/crossfi-mainnet.json:32]: Remove these two lines as well, not acceptable here

**Comment by MortelliGaming** [comment]
i dont understand why its not building now?

**Comment by editaahn** [comment]
Should offer `nodeProvider` information.
You can test the file validation in your local environment. Github Action let you know what is wrong one by one. Check the details on README please.

**Comment by MortelliGaming** [comment]
i open new one with image

**Pull request closed without merging**